src/components/org/OrgAdmin/common/Modal/index.tsx

```diff
@@ -0,0 +1,44 @@
+import { IconInfoCircle } from '@sopt-makers/icons';
+import { MouseEvent } from 'react';
+
+import {
+  StInfoCloseButton,
+  StInfoDescription,
+  StInfoImg,
+  StInfoSubDescription,
+  StInfoTitle,
+  StInfoWrapper,
+} from './style';
+
+interface ModalProps {
+  title: string;
+  description: string;
+  subDescription: string;
+  imgSrc: string;
+  isInfoVisible: boolean;
+  onInfoToggle: (e: MouseEvent<HTMLButtonElement>) => void;
+}
+
+const Modal = ({
+  title,
+  description,
+  subDescription,
+  imgSrc,
+  isInfoVisible,
+  onInfoToggle,
+}: ModalProps) => {
+  return (
+    <StInfoWrapper isVisible={isInfoVisible}>
+      <StInfoTitle>
+        <IconInfoCircle />
+        <span>{title}</span>
+        <StInfoCloseButton onClick={onInfoToggle}>&#10005;</StInfoCloseButton>
```

Comment:
```suggestion
<StInfoCloseButton onClick={onInfoToggle} aria-label="Close Modal">
```

aria-label 속성을 둬서 닫기 버튼의 역할을 명확히 표현하면 좋을 거 같습니다 ◡̈